test/framework/options.py

```diff
@@ -2942,6 +2942,21 @@ def test_review_pr(self):
         regex = re.compile(r"^Comparing gzip-1.10-\S* with gzip-1.10-")
         self.assertTrue(regex.search(txt), "Pattern '%s' not found in: %s" % (regex.pattern, txt))
 
+        self.mock_stdout(True)
+        self.mock_stderr(True)
+        # closed, unlabeled PR for gzip 1.2.8 easyconfig,
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/5365
```

Comment:
Hmm, not sure if this is a good idea, since if some maintainer thinks "hey, this PR never got properly labeled, let me fix that", this test will be broken...

Can we do better, by somehow ignoring the labels that were set, to ensure the test will never fail?

Maybe even go as far as supporting an `--ignore-pr-labels` option?

Reply:
is there another use case for `--ignore-pr-labels`?

what about something like

```diff
diff --git a/easybuild/framework/easyconfig/tools.py b/easybuild/framework/easyconfig/tools.py
index 2df6450..0ae7ea0 100644
--- a/easybuild/framework/easyconfig/tools.py
+++ b/easybuild/framework/easyconfig/tools.py
@@ -506,7 +506,7 @@ def find_related_easyconfigs(path, ec):
     return sorted(res)
 
 
-def review_pr(paths=None, pr=None, colored=True, branch='develop'):
+def review_pr(paths=None, pr=None, colored=True, branch='develop', testing=False):
     """
     Print multi-diff overview between specified easyconfigs or PR and specified branch.
     :param pr: pull request number in easybuild-easyconfigs repo to review
@@ -549,7 +549,7 @@ def review_pr(paths=None, pr=None, colored=True, branch='develop'):
     pr_target_account = build_option('pr_target_account')
     github_user = build_option('github_user')
     pr_data, _ = fetch_pr_data(pr, pr_target_account, pr_target_repo, github_user)
-    pr_labels = [label['name'] for label in pr_data['labels']]
+    pr_labels = [label['name'] for label in pr_data['labels']] if not testing else []
 
     expected_labels = det_pr_labels(file_info, pr_target_repo)
 
diff --git a/easybuild/main.py b/easybuild/main.py
index 415321d..8bd9ed5 100644
--- a/easybuild/main.py
+++ b/easybuild/main.py
@@ -247,7 +247,7 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
         merge_pr(options.merge_pr)
 
     elif options.review_pr:
-        print(review_pr(pr=options.review_pr, colored=use_color(options.color)))
+        print(review_pr(pr=options.review_pr, colored=use_color(options.color), testing=testing))
 
     elif options.list_installed_software:
         detailed = options.list_installed_software == 'detailed'
diff --git a/test/framework/options.py b/test/framework/options.py
index 0230bf5..c76841c 100644
--- a/test/framework/options.py
+++ b/test/framework/options.py
@@ -2944,14 +2944,14 @@ class CommandLineOptionsTest(EnhancedTestCase):
 
         self.mock_stdout(True)
         self.mock_stderr(True)
-        # closed, unlabeled PR for gzip 1.2.8 easyconfig,
+        # closed PR for gzip 1.2.8 easyconfig,
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/5365
         args = [
             '--color=never',
             '--github-user=%s' % GITHUB_TEST_ACCOUNT,
             '--review-pr=5365',
         ]
-        self.eb_main(args, raise_error=True)
+        self.eb_main(args, raise_error=True, testing=True)
         txt = self.get_stdout()
         self.mock_stdout(False)
         self.mock_stderr(False)
```

?

Reply:
@migueldiascosta That's acceptable imho, just add a comment in `review_pr` to explain why the `testing` named argument is there, and where it is used.